### DISCUSSION TITLE
question about obhistory from w1.weather.gov

### DISCUSSION THREAD

**Discussion by jsvnsk** [weather-gov/api#666] [Q&A]
Hi -- I just wanted to check if this forum is the right place to ask a question about data from
https://w1.weather.gov/data/obhistory/<stationId>
If not, where can I make an inquiry about that data?
Thanks!

**Comment by dwhitemv25** [comment]
What's the question? Looks like it is providing translated METAR reports.

**Reply by jsvnsk** [reply]
On October 16 and 17, some stations (mostly East of the Rockies) were reporting results in UTC instead of local time. Interestingly, the issue seems to be resolved today so perhaps it was just an intermittent glitch.
